README.md

```diff
@@ -1,38 +1,53 @@
 # ctb-backend
 
-Jak to uruchomić?
+## How to run
```

Comment:
I suggest adding *what* we want to run and *where*, i.e.: "How to run the server in a Docker container?".